.circleci/config.yml

```diff
@@ -367,7 +367,7 @@ jobs:
       - run:
           name: build the web app
           working_directory: ~/project/web
-          command: API_URL="${<< parameters.env >>_WEB_API_URL}" npm run build
+          command: API_URL="${<< parameters.env >>_WEB_API_URL}" OKTA_DOMAIN="${OKTA_DOMAIN}" npm run build
```

Comment:
This would be easier to read with the following syntax:

```yaml
command: |
  API_URL="${<< parameters.env >>_WEB_API_URL}" \
  OKTA_DOMAIN="${OKTA_DOMAIN}" \
  npm run build
```